src/utils.js

```diff
@@ -58,3 +53,73 @@ function getHashParams (link) {
   hashParamsObj.key = Buffer.from(hashParamsObj.key, 'base64')
   return hashParamsObj
 }
+
+/**
+   * Get a value
+   * @param {Object} db - The hyperDB instance
+   * @param {CryptoKey} enckey - The enc/dec AES key
+   * @param {string} key - The key name
+   * @returns {Promise}
+   */
+const get = async (db, encKey, key) => {
+  if (!db) throw new MasqError(ERRORS.NODB)
+  if (!encKey) throw new MasqError(ERRORS.NOENCRYPTIONKEY)
+  const node = await db.getAsync(key)
+  if (!node) return null
+  const dec = await decrypt(encKey, node.value)
+  return dec
+}
+
+/**
+   * Put a new value in the db
+   * @param {Object} db - The hyperDB instance
+   * @param {CryptoKey} enckey - The enc/dec AES key
+   * @param {string} key - The key name
+   * @param {any} value - The value to insert
+   * @returns {Promise}
+   */
+const put = async (db, encKey, key, value) => {
+  if (!db) throw new MasqError(ERRORS.NODB)
```

Comment:
maybe check the type oh db object: `if (!(this instanceof HyperDB))`

Reply:
done